crates/cli/src/subcommands/delete.rs

```diff
@@ -10,19 +11,10 @@ pub fn cli() -> clap::Command {
                 .required(true)
                 .help("The domain or address of the database to delete"),
         )
-        .arg(
-            Arg::new("identity")
-                .long("identity")
-                .short('i')
-                .help("The identity to use for deleting this database")
-                .long_help("The identity to use for deleting this database. If no identity is provided, the default one will be used."),
-        )
-        .arg(
-            Arg::new("server")
-                .long("server")
-                .short('s')
-                .help("The nickname, host name or URL of the server hosting the database")
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.